Content.Tests/DMProject/Tests/Math/minmax.dm

```diff
@@ -0,0 +1,16 @@
+/proc/RunTest()
+	ASSERT(min(1,2,3) == 1)
+	ASSERT(min(null, 1) == null)
+	ASSERT(min(null, -1) == -1)
+	ASSERT(min(1, null) == null)
+	ASSERT(min(0,null)==null)
```

Comment:
This test does not test `min(null, 0)`. In BYOND this gives `0` but this PR is giving `null`.
```suggestion
	ASSERT(min(0,null)==null)
	ASSERT(min(null, 0) == 0)
```